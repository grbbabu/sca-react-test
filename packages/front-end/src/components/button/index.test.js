import React from 'react';
import renderer from 'react-test-renderer';
import Button from './index';

it('renders correctly when there are no input to the form', () => {
    const tree = renderer
        .create(<Button type="button" label="Submit" />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
