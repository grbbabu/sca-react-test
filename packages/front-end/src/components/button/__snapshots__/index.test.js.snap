// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no input to the form 1`] = `
<button
  className="button"
  disabled={false}
  onClick={null}
  type="button"
>
  Submit
</button>
`;
